scripts/install/deploy/install_db.sh

```diff
@@ -1,8 +1,16 @@
 #!/bin/sh
 
 # @todo #299:30min Continue the deploy algorithm. We need to create an
```

Comment:
@paulodamaso You should change the code of the puzzle, otherwise you are not removing it